roborock/code_mappings.py

```diff
@@ -438,6 +458,17 @@ class RoborockMopIntensityS8MaxVUltra(RoborockMopIntensityCode):
     custom_water_flow = 207
 
 
+class RoborockMopIntensitySaros10R(RoborockMopIntensityCode):
+    off = 200
+    low = 201
+    medium = 202
+    high = 203
+    custom = 204
+    extreme = 250
+    vac_followed_by_mop = 235
```

Comment:
Hi, I am getting similar codes on my Saros 10 (non R) but this one I don't get, are you sure it's also under the `water_box_mode` field?

Reply:
Yes, I checked quite throughly.
It happens when selecting the mode "vac followed by mop" instead of "vacuum & mop", then don't change any of the settings like water flow, otherwise it goes back to "vacuum & mop"

Reply:
It will go back to the "vacuum & mop" and apply the "extreme" setting for the water flow.
(I think roborock could have made this better or more clear in their UI of the app...)

Reply:
I don't know what it is for my Saros 10, but 235 doesn't work and throws an error.